python/hail/py3_compat.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+if sys.version_info > (3,):
+    xrange = range
+    long = int
+    unicode = str
```

Comment:
I think this logic should be moved inside the check module - this is the only place they're needed explicitly, I think.